staging/src/k8s.io/client-go/tools/reference/ref.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package ref
+package reference
```

Comment:
@sttts could you move it to client-go/util instead? In my mind, tools/ are for larger features like informers ect. And when k8s.io/pkg is ready, util/ should be moved there.